packages/opentelemetry-instrumentation-bedrock/opentelemetry/instrumentation/bedrock/__init__.py

```diff
@@ -361,6 +380,17 @@ def _set_llama_span_attributes(span, request_body, response_body):
         span, SpanAttributes.LLM_REQUEST_MAX_TOKENS, request_body.get("max_gen_len")
     )
 
+    if Config.enrich_token_usage and response_body.get("prompt_token_count") is not None and response_body.get("generation_token_count") is not None:
```

Comment:
same here, no need for the `Config.enrich_token_usage`

Reply:
Thanks for the pointers, Nir! I've addressed the lint issue and consolidated the attribute addition logic for usage into a function call common for ai21, anthropic, and meta. I also implemented similar logic for Cohere based on their API documentation for Command R and related models. However, when I ran a local test, I noticed that the model did not return token counts as the documentation suggested it would. So I've wrapped that logic in a try-catch but still kept it here (O(1) hit at worst -- not much to lose).